addons/ofxiOS/src/video/ofxiOSVideoPlayer.mm

```diff
@@ -14,7 +14,7 @@
     bResetPixels = false;
     bUpdatePixels = false;
     bUpdateTexture = false;
-    bTextureCacheSupported = (CVOpenGLESTextureCacheCreate != NULL);
+    bTextureCacheSupported = (&CVOpenGLESTextureCacheCreate != NULL);
```

Comment:
This comparison was flagged by Xcode as always being false. Have to compare the reference.